graylog2-server/src/main/java/org/graylog2/utilities/ConfigurationMapConverter.java

```diff
@@ -62,6 +65,13 @@
                 case "boolean":
                     value = "true".equalsIgnoreCase(String.valueOf(entry.getValue()));
                     break;
+                case "list":
+                    final List<?> valueList = entry.getValue() == null ? Collections.emptyList() : (List<?>) entry.getValue();
```

Comment:
I guess this can be `final List<String> valueList` since you are converting everything to a string.